boilerplate/app/models/root-store/setup-root-store.ts

```diff
@@ -10,6 +10,27 @@ import { Api } from "../../services/api"
  */
 const ROOT_STATE_STORAGE_KEY = "root"
 
+/**
```

Comment:
Cleaning up some lint rule failures

Reply:
Post rebase I think this may be wrong? It looks like this entire function got added but during implementation of this PR I just did cleanup.